recipe/meta.yaml

```
@@ -35,6 +35,7 @@ requirements:
 
 test:
   source_files:
+    - examples
```

```suggestion
    - examples/miniforge
```

I think we only need this one.